### PR TITLE
Refactored facets to utilize the combined serializer

### PR DESCRIPTION
**Pull request by jaredgalanis**
Co-Authored-By: John Abrahams <jabrah23@gmail.com>

### CODE REVIEW

File: js/packages/glimmer-idc/src/components/facets.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import Component, { hbs } from '@glimmerx/component';
 
 export default class Facets extends Component {
```
Comment (jabrah) [inline js/packages/glimmer-idc/src/components/facets.ts:3]: Is this file needed still?

**Review by jabrah — APPROVED** [review]
Looks good, just need to re-build to bring in changes from `develop`

File: js/packages/glimmer-idc/src/components/facets.ts (hunk 1 above)
Comment (jaredgalanis) [reply js/packages/glimmer-idc/src/components/facets.ts]: no def not. good catch.
